intrastat_product/models/intrastat_product_declaration.py

```diff
@@ -477,11 +477,12 @@ def _prepare_invoice_domain(self):
         start_date = date(int(self.year), int(self.month), 1)
         end_date = start_date + relativedelta(day=1, months=+1, days=-1)
         domain = [
-            ("date", ">=", start_date),
-            ("date", "<=", end_date),
+            ("invoice_date", ">=", start_date),
```

Comment:
I think the accounting date is used here on purpose

Reply:
In version 12, it was date_invoice

Reply:
Ok then